src/parse.c

```diff
@@ -856,6 +856,9 @@ static int hyper_parse_interface(struct hyper_interface *iface,
 			}
 			ipaddr_oldf->mask = (json_token_str(json, &toks[++i]));
 			dbg_pr(stdout, "net mask is %s\n", ipaddr_oldf->mask);
+		}  else if (json_token_streq(json, &toks[i], "mtu")) {
+			iface->mtu = (json_token_int(json, &toks[++i]));
+			fprintf(stdout, "mtu is %d\n", iface->mtu);
```

Comment:
whats the convention? should the fprintfs to stderr added in net.c also be replaced by dbg_pr?